default/stringtables/en.txt

```diff
@@ -14633,7 +14633,12 @@ MONSTER_NAMES
 [[SHIP_NAME_UNSORTED_MYTHOLOGY]]
 [[SHIP_NAME_DEVS]]'''
 
-
+AI_FIRST_TURN_GREETING
+'''Ave, Human, morituri te salutant!
+Get out of our universe!
+Prepare to die.
+8008135
```

Comment:
Are you objecting specifically to the "8008135" line, or all those additional greetings in general?